### PR TITLE
[WIP] - Demo 2

### PR DESCRIPTION
**Pull request by barbaracassani**
Would be squashed at merge time -- if/when we merge it.

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -40,6 +40,9 @@ module "submit-your-appeal-frontend" {
     EXTERNAL_HOSTNAME            = "${var.env != "preview" ? var.sya_hostname : "${var.deployment_namespace}-sscs-tribunals-frontend-${var.env}.service.${local.aseName}.internal"}"
     HPKP_SHA256                  = "${data.vault_generic_secret.hpkp_sya_sha_1.data["value"]}"
     HPKP_SHA256_BACKUP           = "${data.vault_generic_secret.hpkp_sya_sha_2.data["value"]}"
+    EVIDENCE_UPLOAD_ENABLED      = ""
+    UPLOAD_EVIDENCE_URL          = "${local.ApiUrl}/evidence/upload"
+    LOG_LEVEL                    = "debug"
```
Comment (iguannago) [inline infrastructure/main.tf]: This is the kind of things I am concerned about. Infrastructure changes done in the main configuration that we don't want to bring in yet.

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "@hmcts/look-and-feel": "1.18.0",
     "@hmcts/nodejs-healthcheck": "^1.4.3",
     "@hmcts/nodejs-logging": "^3.0.0",
-    "@hmcts/one-per-page": "3.4.1",
+    "@hmcts/one-per-page": "3.4.3",
```
Comment (iguannago) [inline package.json]: another change that is not related to Evidence Upload and we don't really know why you want to bring in master. Risky!

File: infrastructure/main.tf (hunk 1 above)
Comment (barbaracassani) [reply infrastructure/main.tf]: We can very easily remove this. I shall remove.

File: package.json (hunk 2 above)
Comment (barbaracassani) [reply package.json]: oh dear :) I shall remove :)
